crates/rslint_core/src/groups/errors/no_new_symbol.rs

```diff
@@ -0,0 +1,64 @@
+use crate::rule_prelude::*;
+use ast::*;
+use SyntaxKind::*;
+
+declare_lint! {
+    /**
+    `Symbol` shouldn't be constructed using `new` keyword, instead
+    it should be called as a function.
```

Comment:
```suggestion
    Disallow constructing `Symbol` using `new`.

    `Symbol` shouldn't be constructed using `new` keyword since it results in a `TypeError`, instead
    it should be called as a function.
```
small docs nit